src/docfx/config/Config.cs

```diff
@@ -23,9 +26,14 @@ internal sealed class Config
         public readonly string Name = string.Empty;
 
         /// <summary>
-        /// Gets the contents that are managed by this docset.
+        /// Gets the file glob patterns included by the docset.
+        /// </summary>
+        public readonly string[] Content = s_defaultContentInclude;
+
+        /// <summary>
+        /// Gets the file glob patterns excluded from this docset.
         /// </summary>
-        public readonly FileConfig Content = new FileConfig();
+        public readonly string[] ContentExclude = s_defaultContentExclude;
```

Comment:
Naming: what about `files` and `exclude` as in v2? Jekyll also has a `exclude` at config's top level.

Reply:
sure